<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>driverStatusCheckClass</class>
 <widget class="QMainWindow" name="driverStatusCheckClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>448</width>
    <height>194</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>driverStatusCheck</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="oneClass">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>80</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>one-class SVM</string>
    </property>
   </widget>
   <widget class="QPushButton" name="twoClass">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>80</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>two-class SVM</string>
    </property>
   </widget>
   <widget class="QPushButton" name="titleText">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>10</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;</string>
    </property>
    <property name="text">
     <string>choose SVM type</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>448</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="driverstatuscheck.qrc"/>
 </resources>
 <connections/>
</ui>
